client/components/tinymce/plugins/wpcom-track-paste/plugin.js

```diff
@@ -46,39 +42,38 @@ function pasteTrack( editor ) {
 		return SOURCE_UNKNOWN;
 	}
 
-	function flattenArray( orgTypes ) {
-		// As per https://html.spec.whatwg.org/multipage/interaction.html#datatransfer
-		// types should be an array containing the available formats that were set on copy.
-		// Some browsers, as Firefox, return a DOMStringList instead, see
-		// https://developer.mozilla.org/en-US/docs/Web/API/DOMStringList
-		let types = [];
-		if ( orgTypes.contains ) {
-			types = Array.from( orgTypes );
-		} else {
-			types = orgTypes.slice();
-		}
-		return types.reduce( ( a, b ) => {
-			return a + ', ' + b;
-		} );
+	function toString( types ) {
+		// types should be an array. Some browsers, as Firefox, return a DOMStringList instead.
+		// We need coverage for thoses cases, hence the Array.from conversion.
+		//
+		// https://html.spec.whatwg.org/multipage/interaction.html#datatransfer
+		// DOMStringListhttps://developer.mozilla.org/en-US/docs/Web/API/DOMStringList
```

Comment:
this would be great as a JSDoc comment instead of an inline comment

also, this function doesn't depend on `this` so maybe we can move it out of the encapsulating function and into the module scope.

I like the direction of `toString` but I wonder if that's most appropriate since we already have `toString` as a standard function with standard expectations (it _might_ be the right one - I just want us to pause and make sure it is)